conjur/api/client.py

```diff
@@ -130,20 +135,28 @@ def __init__(self,
                             **loaded_config)
             self._api.login(login_id, password)
         else:
-            try:
-                credentials = CredentialsFromFile(DEFAULT_NETRC_FILE)
-                loaded_netrc = credentials.load(loaded_config['url'])
-            except netrc.NetrcParseError as netrc_error:
-                raise Exception("Error: netrc is in an invalid format. "
-                                f"Reason: {netrc_error}") from netrc_error
-            except Exception as exception:
-                # pylint: disable=line-too-long
-                raise RuntimeError("Unable to authenticate with Conjur. Please log in and try again.") from exception
+            if keyring.get_keyring().name in SUPPORTED_BACKENDS:
+                logging.debug(f"Attempting to fetch credentials from keyring '{keyring.get_keyring().name}'")
+                credentials = CredentialsFromKeystore()
+                loaded_credentials = credentials.load(loaded_config['url'])
```

Comment:
we discussed previously that I didnt want to change this because it has cascading effects especially since we use kwargs in API initialization below
```
            self._api = Api(...
                            **loaded_config)
```